node/_process/streams.mjs

```diff
@@ -103,7 +103,7 @@ export const stdout = stdio.stdout = createWritableStdioStream(
 
 /** https://nodejs.org/api/process.html#process_process_stdin */
 export const stdin = stdio.stdin = new Readable({
-  highWaterMark: 0,
+  highWaterMark: Deno.isatty?.(Deno.stdin.rid) ? 0 : 64 * 1024,
```

Comment:
`Deno.stdin` also needs to be guarded by `?` to make this compatible with Deno Deploy
```suggestion
  highWaterMark: Deno.isatty?.(Deno.stdin?.rid) ? 0 : 64 * 1024,
```